### DISCUSSION TITLE
1.0.0 Beta - Add Post to view

### DISCUSSION THREAD

**Discussion by Xbanjwk17** [Xbanjwk17/FreedomBack#10] [Announcements]
Pre-released you can post view.

<hr /><em>This discussion was created from the release <a href='https://github.com/Xbanjwk17/FreedomBack/releases/tag/1.0.0b'>1.0.0 Beta - Add Post to view</a>.</em>

**Comment by CodingNepal14** [comment]
Hehe, pre-released is turn on.
